doc/api/async_hooks.md

```diff
@@ -271,11 +271,16 @@ created, while `triggerAsyncId` shows *why* a resource was created.
 The following is a simple demonstration of `triggerAsyncId`:
 
 ```js
+const async_hooks = require('async_hooks');
+const fs = require('fs');
```

Comment:
Why go CJS rather than ESM? 🙈 
Maybe it's better to leave this part out to avoid debating over it.

Reply:
@aduh95 Agreed! Updated the code to use the modules directly. :)